src/app/qgsoptions.cpp

```diff
@@ -2300,3 +2300,25 @@ void QgsOptions::scaleItemChanged( QListWidgetItem *changedScaleItem )
   addScaleToScaleList( changedScaleItem );
   mListGlobalScales->setCurrentItem( changedScaleItem );
 }
+
+double QgsOptions::getZoomFactorValue( QSpinBox *zoomFactorSpinBox )
```

Comment:
I don't think it's required to pass this as a parameter.

This is a class member variable which is available in here anyway. Just use `spinZoomFactor` directly.